tests/helpers/start-app.js

```diff
@@ -1,5 +1,5 @@
-import Ember from 'ember'
-const {merge, run} = Ember
+import {merge} from '@ember/polyfills'
```

Comment:
This wouldn't come off of `Object`? If not, why in the previous file does it not come off of `@ember/polyfills` instead of `Object`?

Reply:
I double checked this import to make sure and it is the correct module.